tools/chrome-debug-extension/manifest.json

```diff
@@ -0,0 +1,44 @@
+{
+    "name": "Telemetry Viewer",
+    "short_name": "Telemetry Viewer",
+    "description": "A browser extension that provides a real time view of what's happening in Application Insights including what telemetry is being logged by the web application",
+    "version": "0.2.0",
+    "version_name": "0.2.0",
+    "manifest_version": 2,
+    "icons": {
+        "16": "images/icon-16.png",
+        "128": "images/icon-128.png"
+    },
+    "permissions": [
+        "<all_urls>",
+        "activeTab",
+        "webRequest"
+    ],
+    "background": {
+        "scripts": [
+            "scripts/background.js"
+        ]
+    },
+    "browser_action": {
+        "default_icon": {
+            "19": "images/icon-19.png",
+            "38": "images/icon-38.png"
+        },
+        "default_title": "Telemetry Viewer"
+    },
+    "web_accessible_resources": [
+        "scripts/pageHelper.js",
+        "scripts/pageHelper.min.js",
+        "scripts/pageHelper.min.js.map"
+    ],
+    "content_scripts": [
+        {
+            "matches": [
+                "<all_urls>"
+            ],
+            "js": [
+                "scripts/contentLoad.min.js"
```

Comment:
Min is for all builds to minimize the amount of code downloaded.

Yes, for local developers (of the tool) this will cause the file to be "minified" there are 2 options here
- Comment out the minification step for the build (rollup.config.js) so it doesn't generate the min file
- Or use the files directly (not via loading the extension) `npm run serve` and load the browser/pages/popup.html file, this will then load the map files (which are generated)... Map files are not loaded by chrome for extensions.

The 2 files that are minified by default are relatively small and I don't expect them to be changed a lot. I left the main popup.js unminified for development purposes as this really only affects the popup itself.

Reply:
Can we we not minify debug builds, and minify everything for production builds?  It looks like that's what we're doing already in some of the sections of the rollup.config.js.

Reply:
We don't have "debug" builds we only have a single which generates artifacts and at the moment we are only generating a single pre-packaged zip file (which will ALWAYS effectively be production as this is what we release).

Even for the SDK everything is production (minified and unminified) as the end-consumer chooses what to consume (via the CDN) and for NPM users nothing is minified.

Reply:
And for this file we are talking about this code only
```
chrome.runtime.sendMessage({ id: MessageType.ContentLoad });

/**
 * Listen to page events
 */
window.addEventListener("message", (event) => {
    if (event.source != window) {
        return;
    }

    chrome.runtime.sendMessage(event.data);
}, false);
```